backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/action/CreateVolumeContainerCommandParameters.java

```diff
@@ -13,6 +14,8 @@ public class CreateVolumeContainerCommandParameters extends StorageJobCommandPar
     private Long initialSize;
     private boolean legal = true;
 
+    private DiskBackup backup;
```

Comment:
If Shani agrees with my comment at
https://github.com/oVirt/ovirt-engine/pull/398/files/8c40c0cdada4c8c57140e2d6788a5bc372d1f07b#r882853060
worth changing it here as well to be consistent.

Reply:
On my patch, Benny said it doesn't necessary :)